src/laserscan_to_pointcloud_node.cpp

```diff
@@ -49,7 +49,11 @@
 #include <utility>
 
 #include "sensor_msgs/point_cloud2_iterator.hpp"
-#include "tf2_sensor_msgs/tf2_sensor_msgs.h"
+#ifdef USE_TF2_SENSOR_MSGS_DEPRECATED_HEADER
+#include <tf2_sensor_msgs/tf2_sensor_msgs.h>
```

Comment:
@soblin [NITS]: I think it is better to keep double quotes unless you have any specific reason to change them to brackets.